packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -228,22 +229,11 @@ class EditorPage {
 
 		const htmlContentView = await this.getTextViewForHtmlViewContent();
 
-		if ( isAndroid() ) {
-			// Attention! On Android `.type()` replaces the content of htmlContentView instead of appending
-			// contrary to what iOS is doing. On Android tried calling `driver.pressKeycode( 279 ) // KEYCODE_PASTE`
-			// before to paste, but for some reason it didn't work on GitHub Actions but worked only on Sauce Labs
-			await htmlContentView.type( html );
-		} else {
-			await htmlContentView.click();
-			await doubleTap( this.driver, htmlContentView );
-			// Sometimes double tap is not enough for paste menu to appear, so we also long press.
-			await longPressMiddleOfElement( this.driver, htmlContentView );
+		await htmlContentView.click();
+		await doubleTap( this.driver, htmlContentView );
+		// Sometimes double tap is not enough for paste menu to appear, so we also long press.
 
```

Comment:
The double tap comment should likely be removed or relocated to within `tapPasteAboveElement`. Currently it is a bit cryptic and separated by a confusing empty line.

Reply:
Nice catch! I'll remove it instead of moving it to `tapPasteAboveElement` 👍